src/components/base/JInputField/jInputField.m.scss

```diff
@@ -93,35 +86,40 @@
 .white-indicator {
   .wrap {
     background-color: $white;
+    box-shadow: $normal-shadow;
+
+    &:hover {
+      box-shadow: $hover-shadow;
+    }
   }
 
   .input {
     color: $main-black;
   }
 
   .label {
-    color: $main-black-40;
-  }
-
-  &:hover .input::placeholder,
-  &:hover .label {
     color: $main-black-30;
   }
 
-  &.error .label {
+  .error .label {
     color: $main-red;
   }
 
-  &.disabled {
-    background: $main-black-5;
+  .disabled {
+    box-shadow: none;
+    background: $white;
+
+    &:hover {
+      box-shadow: none;
+    }
 
     .label {
-      user-select: none;
       cursor: default;
+      color: $main-black-40;
```

Comment:
but it must be `$main-black-30`

Reply:
please recheck figma screens

Reply:
Checked and fixed
value + disabled - 30%
just disabled with value - 40%